uber/models.py

```diff
@@ -1338,6 +1361,13 @@ def requested(self, department):
     def assigned_to(self, department):
         return int(department or 0) in self.assigned_depts_ints
 
+    def trusted_in(self, department):
+        return int(department or 0) in self.trusted_depts_ints
+
```

Comment:
The more I look at this the more I think I would name it ``.trusted_anywhere`` but that's a minor naming preference and not something you need to change unless the suggestion makes you jump up and say "OF COURSE!"